recirq/qcqmc/hamiltonian.py

```diff
@@ -0,0 +1,319 @@
+from pathlib import Path
+from typing import Tuple, Union
+
+import attrs
+import fqe
+import h5py
+import numpy as np
+import openfermion as of
+from fqe.hamiltonians.restricted_hamiltonian import RestrictedHamiltonian
+from fqe.openfermion_utils import integrals_to_fqe_restricted
+from openfermionpyscf._run_pyscf import compute_integrals  # type: ignore
+from pyscf import ao2mo, fci, gto, scf
+
+from recirq.qcqmc.config import OUTDIRS
+from recirq.qcqmc.data import Data, Params, get_integrals_path
+
+
+@attrs.frozen
+class LoadFromFileHamiltonianParams(Params):
```

Comment:
This name is awkward.  Load is a verb, so this seems like an action, but the class holds parameters?
I would rename to HamiltonianFileParameters

Reply:
Done